plugins/UM3NetworkPrinting/resources/qml/MonitorPrinterCard.qml

```diff
@@ -359,6 +359,10 @@ Item
                     {
                         return catalog.i18nc("@label:status", "Idle")
                     }
+                    if (!printer.activePrintJob && printer.state == "pre_print")
+                    {
+                        return catalog.i18nc("@label:status", "Preparing...")
```

Comment:
This re-uses an existing translation so it doesn't break the string freeze for 4.8, in case anyone is wondering.